src/commands/mobile/plugin/sentry.ts

```diff
@@ -0,0 +1,167 @@
+import { copyFiles, parseMobileModuleConfig, updateDynamicImportsAndExports, readConfigFile, writeFile } from '../../../lib/files';
+import { checkProjectValidity, isJsonString, parseAuthorizationToken, parseOrganization } from '../../../lib/utilities';
+import { MOBILE_CLI_COMMANDS, CLI_STATE } from '../../../lib/constants';
+
+const TEMPLATE_FOLDERS = ['sentry'];
+const CUSTOM_ERROR_CODES = [
+  'project-invalid',
+  'missing-template-file',
+  'missing-template-folder',
+  'dependency-install-error',
+];
+
+export default class Sentry extends Command {
+  static description = 'application performance monitoring and error handling support';
+
+  static flags = {
+    help: Flags.help({ char: 'h' }),
+    forceProject: Flags.string({ hidden: true }),
+    skipInstall: Flags.boolean({ hidden: true }),
+  }
+
+  static args = {
+    organization: Args.string({ name: 'organization', description: 'organization of the project' }),
+    project: Args.string({ name: 'project', description: 'project name of of the project' }),
+    authorizationToken: Args.string({ name: 'authorization token', description: 'Unique identifier that will used for connection to the sentry API' }),
+  }
+
+  // override Command class error handler
+  catch(error: Error): Promise<any> {
+    const errorMessage = error.message;
+    const isValidJSON = isJsonString(errorMessage);
+    const parsedError = isValidJSON ? JSON.parse(errorMessage) : {};
+    const customErrorCode = parsedError.code;
+    const customErrorMessage = parsedError.message;
+    const hasCustomErrorCode = customErrorCode !== undefined;
+
+    if (hasCustomErrorCode === false) {
+      // throw cli errors to be handled globally
+      throw errorMessage;
+    }
+
+    // handle errors thrown with known error codes
+    if (CUSTOM_ERROR_CODES.includes(customErrorCode)) {
+      this.log(`${CLI_STATE.Error} ${customErrorMessage}`);
+    } else {
+      throw new Error(customErrorMessage);
+    }
+
+    return Promise.resolve();
+  }
+
+  async run(): Promise<void> {
+    const { flags, args } = await this.parse(Sentry);
+    let projectName = flags.forceProject;
+    const skipInstallStep = flags.skipInstall === true;
+    const hasProjectName = projectName !== undefined;
+    const preInstallCommand = hasProjectName ? `cd ${projectName} &&` : '';
+
+    const projectValidity = checkProjectValidity();
+    const { isValid: isValidProject } = projectValidity;
+    const projectRoot = path.join(process.cwd()).trim();
+
+    if (!projectName) {
+      const rootArray = projectRoot.split('\\');
+      projectName = rootArray[rootArray.length - 1];
+    }
+
+    // block command unless being run within an rdvue project
+    if (isValidProject === false && !hasProjectName) {
+      throw new Error(
+        JSON.stringify({
+          code: 'project-invalid',
+          message: `${MOBILE_CLI_COMMANDS.PluginSentry} command must be run in an existing ${chalk.yellow('rdvue')} project`,
```

Comment:
Update the `rdvue` keywords in the messaging and comment above.

Reply:
@ttaylor92